### PR TITLE
Avoid to inject Experiment code when is not LIVE mode/ Avoid broke th…

### LINKED ISSUE #27584: Edit mode navigation render is broken
### Problem Statement
In edit mode, when navigating to any page, a series of `\n\n\n\n\n` is present in the HTML of the page.

### Steps to Reproduce
1. Log in to the dotCMS admin interface.
2. Go to the "Pages" section.
3. Select any page to edit.
4. View the HTML source code of the page.

### Acceptance Criteria
The issue of unnecessary `\n\n\n\n\n` in the HTML of pages in edit mode should be resolved. The HTML code should be clean and free from any unnecessary line breaks.

### External Links
N/A

### Assumptions & Initiation Needs
- The issue might be related to the backend code or configuration.
- Further research and investigation are required to identify the root cause of the problem.

### Quality Assurance Notes & Workarounds
N/A

### PR DESCRIPTION
**Pull request by freddyDOTCMS**
### Proposed Changes
* Not include Experiment code when is not LIVE mode

https://github.com/dotCMS/core/pull/27653/files#diff-df4c3ee11b473e0f4e91d5be7a6107bf6cdb4ab40bf0661a182dd298e00e5602R302

- Include scape character to the close tag Script no matter if it is write on Upper or Lower case

https://github.com/dotCMS/core/pull/27653/files#diff-1e8134790658b55b7bab7bbc2330bf65b7106fbd1b015415bd83ade6a8eddc7dR91

-

### CODE REVIEW

**Review by victoralfaro-dotcms — APPROVED** [review]

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java [hunk 1]
```diff
@@ -2363,4 +2351,76 @@ public Host getHost(TestHostType testType) {
             return hostMap.get(testType);
         }
     }
+
+    @Test
+    public void renderPageWithScriptAndInnerNavigateMode() throws Exception {
```
Comment (dsilvam) [inline dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java:2356]: Test doc

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java [hunk 2]
```diff
@@ -2363,4 +2351,76 @@ public Host getHost(TestHostType testType) {
...
+        HttpServletRequest mockRequest = new MockSessionRequest(
+                new MockAttributeRequest(
+                        new MockHttpRequestIntegrationTest("localhost", "/").request())
+                        .request())
+                .request();
+
+        when(mockRequest.getRequestURI()).thenReturn(pageEnglishVersion.getURI());
+        when(mockRequest.getParameter("host_id")).thenReturn(host.getIdentifier());
+        mockRequest.setAttribute(com.liferay.portal.util.WebKeys.USER, systemUser);
+        //mockRequest.setAttribute(WebKeys.HTMLPAGE_LANGUAGE, String.valueOf(language.getId()));
```
Comment (dsilvam) [inline dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java:2407]: remove

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java (hunk 1 above)
Comment (freddyDOTCMS) [reply dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java:2362]: done https://github.com/dotCMS/core/pull/27653/commits/e4b0ff30db27892c44e0515577049028ca0083ba#diff-d17da2e37e530dacc5e41e8534442b71efc4ea5a9a7880b03627b61dddb394d6R2355

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java (hunk 2 above)
Comment (freddyDOTCMS) [reply dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java]: done https://github.com/dotCMS/core/pull/27653/commits/e4b0ff30db27892c44e0515577049028ca0083ba#diff-d17da2e37e530dacc5e41e8534442b71efc4ea5a9a7880b03627b61dddb394d6L2407

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java [hunk 3]
```diff
@@ -2352,6 +2352,12 @@ public Host getHost(TestHostType testType) {
         }
     }
 
+    /**
+     * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * When: The render with more than one Script block no matter is it is on upper case or lower case
```
[the remark this answers is not in the document]
Comment (freddyDOTCMS) [reply dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java]: done https://github.com/dotCMS/core/pull/27653/commits/baea3d13ddfb29ca0ecdb4e7870e0c940c70b15d#diff-d17da2e37e530dacc5e41e8534442b71efc4ea5a9a7880b03627b61dddb394d6R2357

File: dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java [hunk 4]
```diff
@@ -2352,6 +2352,12 @@ public Host getHost(TestHostType testType) {
...
+     * Should: Scape de close script tag as follow \</script/>
```
[the remark this answers is not in the document]
Comment (freddyDOTCMS) [reply dotcms-integration/src/test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java]: done https://github.com/dotCMS/core/pull/27653/commits/baea3d13ddfb29ca0ecdb4e7870e0c940c70b15d#diff-d17da2e37e530dacc5e41e8534442b71efc4ea5a9a7880b03627b61dddb394d6R2358

**Review by dsilvam — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Pull request merged**